tests/test_modeling_longformer.py

```diff
@@ -115,6 +115,18 @@ def prepare_config_and_inputs(self):
     def check_loss_output(self, result):
         self.parent.assertListEqual(list(result["loss"].size()), [])
 
+    def create_and_check_attention_mask_determinism(
```

Comment:
There was a bug previously in that running the model without `attention_mask` and with `attention_mask = torch.tensor([0, ..., 0])` did not give the same output.